apps/tabs-storybook-e2e/src/integration/wizard.component.spec.ts

```diff
@@ -0,0 +1,23 @@
+['default', 'modern-light', 'modern-dark'].forEach((theme) => {
+  describe(`tabs-storybook in ${theme} theme`, () => {
+    beforeEach(() =>
+      cy.visit(
+        `/iframe.html?globals=theme:${theme}&id=wizardcomponent-wizard--wizard`
+      )
+    );
+    it('should render the component', () => {
+      cy.get('app-wizard')
+        .should('exist')
+        .should('be.visible')
+        .get('.open-wizard-btn')
+        .should('exist')
+        .should('be.visible')
+        .click()
+        .get('.sky-modal')
+        .should('exist')
+        .should('be.visible')
+        .screenshot(`wizardcomponent-wizard--wizard-${theme}`)
+        .percySnapshot(`wizardcomponent-wizard--wizard-${theme}`);
```

Comment:
Can you add `, { widths: [375, 1280] }` (manually add the widths from our `.percy.json`) so the media query service applies?